spec/requests/locations_controller_spec.rb

```diff
@@ -1594,7 +1625,7 @@
         end
       end
 
-      context "when deactivation date is during a deactivated period" do
+      xcontext "when deactivation date is during a deactivated period" do
```

Comment:
I'm guessing you either forgot to remove or unsnap this context? Unless there's a ticket that you're aware of needing this or something that I'm not aware of :)

Reply:
Good spot! Thanks!